test/Dialect/MSFT/partition.mlir

```diff
@@ -1,39 +1,57 @@
 // RUN: circt-opt %s --msft-partition -verify-diagnostics -split-input-file | FileCheck %s
+// RUN: circt-opt %s --msft-partition --msft-wire-cleanup -verify-diagnostics -split-input-file | FileCheck --check-prefix=CLEANUP %s
 
-msft.module @top {} (%clk : i1) -> () {
+msft.module @top {} (%clk : i1) -> (out1: i2, out2: i2) {
```

Comment:
I moved to `i2` (from `i1`) to differentiate data from clock by the type. Catch more bugs that way.